### DISCUSSION TITLE
Allow specifying which elements hide when using HUD toggle setting

### DISCUSSION THREAD

**Discussion by Devrill** [ppy/osu#19486] [Ideas]
I think osu lazer needs some customization on toggle HUD on which player can customize what he sees on individual "pages"
For example if player doesn't want to completely hide everything during gameplay, but he only wants to see key overlay and hp bar.

Now we can only customize only main HUD "page" and there is no way to see key overlay while hud is disabled, like in stable.

**Comment by nagi-desuuu** [comment]
There's the skin layout editor.
You can just delete everything except the hp bar and voila!
